src/Umbraco.Core/IO/IOHelper.cs

```diff
@@ -13,7 +13,9 @@
 namespace Umbraco.Core.IO
 {
 	public static class IOHelper
-    {
+	{
+	    public static bool IAmUnitTestingSoNeverUseHttpContextEver = false;

```

Comment:
ah of course! I'd like to kill this class, can't remember what has been done in v8 so far, i think it's an injected service but haven't looked at that for a while.